src/features/projectsV2/ProjectSection.tsx

```diff
@@ -0,0 +1,24 @@
+import ProjectSnippet from './components/ProjectSnippet';
+import style from './styles/ProjectSection.module.scss';
+import { useProjects } from './ProjectsContext';
+
+const ProjectSection = () => {
```

Comment:
Minor change, let's call this `ProjectsSection` to indicate that it handles multiple projects.

Would the controls (search/filter etc) also be within this same component?